include/valkey/valkey.h

```diff
@@ -42,14 +43,15 @@
 struct timeval; /* forward declaration */
 typedef SSIZE_T ssize_t;
 #endif
-#include <stdint.h> /* uintXX_t, etc */
-#include "sds.h" /* for sds */
 #include "alloc.h" /* for allocation wrappers */
+#include "sds.h"   /* for sds */
+
+#include <stdint.h> /* uintXX_t, etc */
 
 #define LIBVALKEY_MAJOR 1
 #define LIBVALKEY_MINOR 2
 #define LIBVALKEY_PATCH 0
-#define LIBVALKEY_SONAME 1.2.1-dev
+#define LIBVALKEY_SONAME 1.2.1 - dev
```

Comment:
This is a problem, only seems to trigger failures in the windows builds for some reason

Reply:
Yup, this one is only used on Windows. I'll try putting quotes there just in case we want to use it anywhere else. If that doesn't fix Windows builds, I'll mark this line as ignored by formatter.